### DISCUSSION TITLE
when browsing images using version 6

### DISCUSSION THREAD

**Discussion by Baltimorepc** [LycheeOrg/Lychee#2659] [Q&A] [answered]
When browsing the images and i go back to continue looking is there a way it can take you back to the one you clicked on instead of starting back at the beginning.

**Comment by ildyria** [comment]
yes I am aware.
That is because we do not record the Y position of the scroll (yet) when moving and in vue those count as "page change".
I am pretty sure I know how to fix it, I just haven't had time to do it.

**Reply by ildyria** [reply]
@Baltimorepc I strongly suggest you switch to `latest` or `nightly` tag by the way.
You no longer need to follow `alpha`. :)

**Comment by Baltimorepc** [comment]
Thank you I didn’t not know that I’ll switch back to master 


Sent from Yahoo Mail for iPhone


On Saturday, November 9, 2024, 6:59 AM, Benoît Viguier ***@***.***> wrote:




@Baltimorepc I strongly suggest you switch to latest or nightly tag by the way.
You no longer need to follow alpha. :)

**Comment by Gendra13** [comment]
👍 This would be indeed a nice enhancement to have.
Especially when browsing large albums or in anticipation of the upcoming timeline feature (#2578) where you often scroll through a lot of thumbnails and only enlarge a few interesting images, it currently feels a bit weird to be returned to the beginning of the gallery when you go back.

**Comment by ildyria** [comment]
Timeline still needs a bit of work.
in mean time: https://github.com/LycheeOrg/Lychee/pull/2694

**Comment by ildyria** [comment, marked as answer]
That part has been fixed.
